README.md

```diff
@@ -1,24 +1,86 @@
-# README
+# Budget App
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> The Ruby on Rails Budget App is about building a mobile web application where user can manage the budget: they have a list of transactions associated with a category, so that the user can see how much money they spent and on what.
 
-Things you may want to cover:
+> The project is following data model below: 
+> ![diagram](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/images/erd_diagram.png)
 
-* Ruby version
+## Built With
 
-* System dependencies
+- Ruby
+- Ruby on Rails
+- PostgreSQL
 
-* Configuration
+## Live Demo 
```

Comment:
- [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md) for this project, " `The project should be deployed and accessible online` ", please make sure your project is deployed properly, the pages work as expected. I get this error (`see screenshot`) when I try to sign up on the online version. Please kindly fix the issue.

![oops](https://user-images.githubusercontent.com/62849872/164728328-cd7f3107-70ca-4c23-bf47-df32a17ed716.png)